### PR TITLE
Updated translations for minor attachment editor changes

### PR DESCRIPTION
**Pull request by zabszk**
Updated all languages with Attachment Editor translated.

### CODE REVIEW

File: Translations/pl/AttachmentEditors.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Niestandardowa
-Konfiguracja nr 
+Niestandardowy
+Zestaw {0}.
 ZAPISZ
 WYCZYŚĆ
-Domyślna
-KONFIGURACJE
+Domyślny
+ZESTAWY
```
Comment (MichalPetryka) [inline Translations/pl/AttachmentEditors.txt:6]: Move unrelated changes to a separate PR

File: Translations/English (default)/AttachmentEditors.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Custom
-Preset #
+Preset #{0}
```
Comment (MichalPetryka) [inline Translations/English (default)/AttachmentEditors.txt:2]:
```suggestion
Preset #[index]
```
Use the named formats.

**Review by MichalPetryka — CHANGES_REQUESTED** [review]

File: Translations/pl/AttachmentEditors.txt (hunk 1 above)
Comment (zabszk) [reply Translations/pl/AttachmentEditors.txt:6]: They are not unrelated. This MR reworks AttachmentEditor file, so this is related. These strings are too long and don't fit the in-game workstations. It's pointless to make a fix that doesn't fix anything (strings still don't fit).

File: Translations/de/AttachmentEditors.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Benutzerdefiniert
-Voreinstellung #
+Voreinstellung #{0}
```
Comment (Rnen) [inline Translations/de/AttachmentEditors.txt:2]:
```suggestion
Voreinstellung #[index]
```

File: Translations/es/AttachmentEditors.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 Personalizado
-Preestablecido #
+Preestablecido #{0}
```
Comment (Rnen) [inline Translations/es/AttachmentEditors.txt:2]:
```suggestion
Preestablecido #[index]
```

File: Translations/fr/AttachmentEditors.txt [hunk 5]
```diff
@@ -1,5 +1,5 @@
 Personnalisé
-Préréglage #
+Préréglage #{0}
```
Comment (Rnen) [inline Translations/fr/AttachmentEditors.txt:2]:
```suggestion
Préréglage #[index]
```

File: Translations/pl/AttachmentEditors.txt [hunk 6]
```diff
@@ -1,9 +1,9 @@
-Niestandardowa
-Konfiguracja nr 
+Niestandardowy
+Zestaw {0}.
```
Comment (Rnen) [inline Translations/pl/AttachmentEditors.txt:2]:
```suggestion
Zestaw #[index].
```

File: Translations/th/AttachmentEditors.txt [hunk 7]
```diff
@@ -1,5 +1,5 @@
 คัสตอม
-พรีเซ็ทที่ #
+พรีเซ็ทที่ #{0}
```
Comment (Rnen) [inline Translations/th/AttachmentEditors.txt:2]:
```suggestion
พรีเซ็ทที่ #[index]
```

File: Translations/zh/AttachmentEditors.txt [hunk 8]
```diff
@@ -1,5 +1,5 @@
 自定义
-预设 #
+预设 #{0}
```
Comment (Rnen) [inline Translations/zh/AttachmentEditors.txt:2]:
```suggestion
预设 #[index]
```

**Review by Rnen — APPROVED** [review]

**Pull request merged**